tsuru/client/auth.go

```diff
@@ -299,89 +299,89 @@ func (c *TeamInfo) Run(context *cmd.Context, client *cmd.Client) error {
 	if err != nil {
 		return err
 	}
-	if resp.StatusCode == http.StatusOK {
-		defer resp.Body.Close()
-		b, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			return err
-		}
-		var team struct {
-			Name  string        `json:"name"`
-			Users []cmd.APIUser `json:"users"`
-			Pools []Pool        `json:"pools"`
-			Apps  []app         `json:"apps"`
-		}
-		err = json.Unmarshal(b, &team)
-		if err != nil {
-			return err
-		}
-		format := `Team: {{.Name}}
+	if resp.StatusCode != http.StatusOK {
+		return nil
+	}
+	defer resp.Body.Close()
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
+	var team struct {
```

Comment:
Hi @guiferpa it would be better if this was declared as a type outside this method :D

Reply:
Sorry I used this very bad practice and I not looked at.